server/images.go

```diff
@@ -690,7 +690,7 @@ func CopyModel(src, dst model.Name) error {
 }
 
 func deleteUnusedLayers(deleteMap map[string]struct{}) error {
-	manifests, err := Manifests()
+	manifests, err := Manifests(true)
```

Comment:
Maybe this should be false to avoid removing files in a undefined state?

Reply:
This gets called in 2 contexts:  PruneLayers, and PullModel.  PruneLayers is already protected and wont be called when there's a corrupt manifest.

PullModel is more nuanced.  deleteUnusedLayers is called with just the layers that are being orphaned by the updated manifest.  I worry that we might make things worse if we stop cleaning up as we pull new manifests.

I'll add a comment explaining the rationale.